regulations/static/regulations/css/less/module/drawer-content.less

```diff
@@ -119,7 +119,7 @@ This contains all of the styles specific to the history
     .effective-label {
         padding: 0 15px;
         .font-regular;
-        color: @dark_gray;
+        color: #666666;
```

Comment:
I was a little torn - you can see that @dark_gray and @dark_text are both being used to denote gray color on text.  They are also used in other contexts so didn't want to just change the variable value.  Adding yet another variable didn't seem like it would be helpful - but I see your point.   Create a more generic variable called something like @drawer_color_gray with a value of #666666 and use it instead?